content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -20,7 +20,7 @@ Setting up a Putaway Rule
 
 In some cases, like for a retail shop storing vegetables and fruits, we have to store products in different locations to maintain product quality. 
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegatable* and *WH/Stock/Fruits*.
+Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegetable* and *WH/Stock/Fruits*.
```

Comment:
Hi! Can we make a quick change to this sentence:

`Let’s suppose there is one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegetables* and *WH/Stock/Fruits*.`

I changed wording that led to a grammatical error.

Reply:
Yes! I was only staring at the single highlighted changes I forgot about the sentence preceding it.

Reply:
@jirocha Do you want to handle this change, or we merge as-is?